libs/blocks/columns/columns.css

```diff
@@ -58,6 +58,12 @@
   letter-spacing: 1px;
 }
 
+@media screen and (min-width:900px) {
```

Comment:
Nit: can we move this below the 600 mq?  
Does this have the same issue as below? This 900vp is rarely used.

Reply:
Columns should match section columns so if we are updating it to 600 then this should be updated too.